apps/guide/content/docs/voice/seeking.mdx

```diff
@@ -2,7 +2,7 @@
 title: Seeking
 ---
 
-Discord.js does not have a built-in method for seeking audio resources. However external libraries such as `prism-media` can be used to tackle this issue.
+discord.js does not have a built-in method for seeking audio resources. However external libraries such as `prism-media` can be used to tackle this issue.
```

Comment:
```suggestion
There is no built-in method for seeking audio resources in discord.js. However external libraries such as `prism-media` can be used to tackle this issue.
```
Because I feel you on the grammar.